contrib/download-frozen-image-v2.sh

```diff
@@ -63,8 +63,8 @@ fetch_blob() {
 			-o "$targetFile" \
 			-D-
 	)"
-	curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
-	if [ "$(echo "$curlHeaders" | awk 'NR == 1 { print $2; exit }')" != '200' ]; then
+        curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
```

Comment:
Looks like this line now uses spaces, not tabs, for indentation; can you fix that? (our shell scripts use tabs for indentation)